lib/engines/bh.js

```diff
@@ -27,9 +28,9 @@ module.exports = function (opts) {
 
         filePath = U.fulfillName(name, opts.ext || this.ext, opts.source);
 
-        // check cache options
-        if (opts.force || options.forceExec || options.forceLoad) {
```

Comment:
They are set in express-bem according to env and some user space code could be broken.

Also if you remove it you couldn't do forcing in render call:
```js
app.render("bundle", {forceLoad: true}, ...);
```